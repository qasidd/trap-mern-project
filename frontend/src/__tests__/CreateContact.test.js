import ContactUs from '../components/Contactsus/createcontact';
import '@testing-library/jest-dom';
import { expect, it } from '@jest/globals';
import { create } from 'react-test-renderer';

it(`It should match the snapshot every time it's rendered`, () => {
    const instance = create(<ContactUs/>).toJSON();
    expect(instance).toMatchSnapshot();
    
})