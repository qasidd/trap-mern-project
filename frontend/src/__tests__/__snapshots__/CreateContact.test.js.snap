// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should match the snapshot every time it's rendered 1`] = `
<div
  className="card"
>
  <div
    className="card-body"
  >
    <div
      className="card-title"
    >
      Contact us Form
    </div>
    <form
      id="contact-form"
      method="post"
      onSubmit={[Function]}
      role="form"
    >
      <input
        className="form-control"
        onChange={[Function]}
        placeholder="Full name"
        required={true}
        type="text"
        value=""
      />
      <input
        className="form-control"
        onChange={[Function]}
        placeholder="Email"
        required={true}
        type="email"
        value=""
      />
      <input
        className="form-control"
        onChange={[Function]}
        placeholder="Subject"
        required={true}
        type="text"
        value=""
      />
      <input
        className="form-control"
        cols="35"
        onChange={[Function]}
        placeholder="Message"
        required={true}
        rows="12"
        type="text"
        value=""
      />
      <br />
      <button
        className="btn btn-outline-success col-md-12"
        type="submit"
      >
        Send
      </button>
      <button
        className="btn btn-outline-success col-md-12"
        onClick={[Function]}
        type="reset"
      >
        Clear
      </button>
    </form>
  </div>
</div>
`;
